src/main/java/graphql/nadel/NadelExecutionHints.java

```diff
@@ -0,0 +1,33 @@
+package graphql.nadel;
+
+public class NadelExecutionHints {
+    private final boolean optimizeExecution;
+
+    private NadelExecutionHints(boolean optimizeExecution) {
+        this.optimizeExecution = optimizeExecution;
+    }
+
+    public static Builder newHints() {
+        return new Builder();
+    }
+
+    public boolean getOptimizeExecutionFlag() {
+        return optimizeExecution;
+    }
```

Comment:
This is a POJO boolean -

convention is to use a `is` prefix

given a `boolean foo` then we have `isFoo` as getter method.

I dont mind `getFoo` in certain cases but there is never a need for a suffix `Flag` on booleans